### DISCUSSION TITLE
使用GPUdocker镜像编译milvus1.x分支编译时错误

### DISCUSSION THREAD

**Discussion by Hiiboom** [milvus-io/milvus#16318] [Q&A and General discussion]
**前言：**
我在使用milvus项目提供的INSTALL.md文档进行Compile Milvus on Docker中的GPUdocker镜像编译
根据由@matrixji 提供的修改代码方法，详情:https://github.com/matrixji/milvus/commit/41ad97d2a1fd1ac6490130d4dc5571b0f294338b
根据以上方法，我将其中的cuda11.4.0替换位11.2.0，除此之外和链接中的修改内容完全相同；
在开始编译前我升级了docker镜像的cuda版本因为更改代码后编译要求cuda为11版本，但官方提供的docker镜像是10.1的cuda

**问题：**
在开始编译后 我使用INSTALL.md中建议的cd ./milvus/core并执行./build.sh -g -t Release，在编译中途出现图中错误:
![8d7444d1c24137d1db44f5863ad9716](https://user-images.githubusercontent.com/58126027/160993601-4d8c3662-5bb3-40ec-919b-bf2bffe5ae4e.png)
如果有知道如何解决的可以告诉我，非常感谢！！！

**Comment by matrixji** [comment]
`memoryType` is not available in Cuda 11, and is already marked as deprecated in Cuda 10(I guess). 
I guess you're using my patch over milvus 0.10.x branch, right?

you could check this code in 1.1 branch:
https://github.com/milvus-io/milvus/blob/de8050d597f01af1abd06f51b98b414c419f8533/core/src/index/thirdparty/faiss/gpu/utils/DeviceUtils.cu#L116-L121

and also check from  faiss's:
https://github.com/facebookresearch/faiss/blob/1806c6af2766548fef8f5addfbadf28f8f23eb21/faiss/gpu/utils/DeviceUtils.cu#L118-L131

**Reply by Hiiboom** [reply]
ok,thank you! I'll try now
